core/jdiameter/impl/pom.xml

```diff
@@ -67,6 +67,16 @@
         <filtering>true</filtering>
       </resource>
     </resources>
+      <plugins>
```

Comment:
Please don't enforce this here, we have added similar @ https://github.com/RestComm/jdiameter/blob/master/pom.xml#L194-L201